src/webgpu/shader/execution/shader_io/compute_builtins.spec.ts

```diff
@@ -176,7 +177,22 @@ g.test('inputs')
     const pass = encoder.beginComputePass();
     pass.setPipeline(pipeline);
     pass.setBindGroup(0, bindGroup);
-    pass.dispatch(t.params.numGroups.x, t.params.numGroups.y, t.params.numGroups.z);
+    if (t.params.dispatch === 'direct') {
```

Comment:
übernit: You can use switch statements on string types in JS / TS (e.g. L49), and I find these slightly more readable.
But this is entirely personal preference, so feel no obligation to change.

Reply:
Done.